Cargo.lock

```diff
@@ -5998,9 +6002,9 @@ dependencies = [
 
 [[package]]
 name = "zip"
-version = "2.1.1"
+version = "2.1.3"
```

Comment:
I wonder why you downgraded all the `symbolic` crates, as the newest version pins this to a lower version, as `2.1.3` has a regression reading archives with >64k files.

Reply:
I have no idea what happened there tbh. I think I added `zip` with the most recent version as a dependency at some point, then it downgraded `symbolic` so it could unify the patch versions?